platformifier/templates/generic/.environment

```diff
@@ -32,9 +32,20 @@ export DATABASE_PORT="$DB_PORT"
 export DATABASE_NAME="$DB_PATH"
 export DATABASE_USERNAME="$DB_USERNAME"
 export DATABASE_PASSWORD="$DB_PASSWORD"
-export DATABASE_SCHEME="$DB_SCHEME"
+export DATABASE_SCHEME="$DB_CONNECTION"
```

Comment:
@akalipetis why did you change `DB_SCHEME` to `DB_CONNECTION` but keep `DB_SCHEME` in the `platformifier/templates/upsun/.environment` file ?

Reply:
It's `DB_CONNECTION` in here - there was a typo previously.